### PR TITLE
Contribution suggestions

### PR DESCRIPTION
**Pull request by nojaf**
@dawedawe could you review the new file `docs/docs/contributors/How Can I Contribute.md` please?
Fixes https://github.com/fsprojects/fantomas/issues/2329.

### CODE REVIEW

File: docs/docs/contributors/How Can I Contribute.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+Besides breaking correctness another kind of bug is that the style of the output might not be what you expect.
+Bugs like this are labelled as `bug (stylistic)`.
+This includes cases where Fantomas breaks one of its own formatting rules or fails to respect one of its settings.
+
+Again, here: scratch your own itch. If something bothers you, the best cure is to try a take a stab at it yourself.
+
+## Good first issues
+
+If wish to solve an issue, but don't know where to start, you can take a look that [good first issue](https://github.com/fsprojects/fantomas/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) list.
```
Comment (dawedawe) [inline docs/docs/contributors/How Can I Contribute.md:38]:
```suggestion
If you wish to solve an issue, but don't know where to start, you can take a look that [good first issue](https://github.com/fsprojects/fantomas/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) list.  
```

File: docs/docs/contributors/How Can I Contribute.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+Again, here: scratch your own itch. If something bothers you, the best cure is to try a take a stab at it yourself.
+
+## Good first issues
+
+If wish to solve an issue, but don't know where to start, you can take a look that [good first issue](https://github.com/fsprojects/fantomas/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) list.  
+These issues are typically easier to pick up and might only require a few small changes to solve them.
+
+In case you want to solve any issue and would like some more guidance to start, you can also just ask this on the GitHub issue.
+They maintainer can give you an additional pointers to get you on your way.
```
Comment (dawedawe) [inline docs/docs/contributors/How Can I Contribute.md:42]:
```suggestion
The maintainer can give you additional pointers to get you on your way.
```

File: docs/docs/contributors/How Can I Contribute.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+Found a typo? Still confused about something? Do you have some knowledge that should totally be documented?  
+Let us know! We really value any contribution to our documentation. The more knowledge is here, the brighter the future of the project.  
+Please do no hesitate here 😊.
+
+You can find some instructions on running the documentation locally in the [.README.md](https://github.com/fsprojects/fantomas/blob/master/docs/.README.md) file in the `docs` folder.  
+The only prerequisite to run the docs, is having a recent local dotnet sdk.
+
+## New releases
+
+Testing out new releases is also a huge way to help us. Spotting on regressions early really helps to fix them early.
```
Comment (dawedawe) [inline docs/docs/contributors/How Can I Contribute.md:104]:
```suggestion
Testing out new releases is also a huge way to help us. Spotting regressions early really helps to fix them early.
```

File: docs/docs/contributors/How Can I Contribute.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+
+### Move to the latest
+
+Always try and stay on the latest version of Fantomas for your day-to-day projects.  
+Updating to the newer version that might only have a couple of fixes might seem insignificant, but it really helps.
+
+### Try alpha's and beta's
```
Comment (dawedawe) [inline docs/docs/contributors/How Can I Contribute.md:111]:
```suggestion
### Try alphas and betas
```

File: docs/docs/contributors/How Can I Contribute.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+
+Fantomas uses the parser from the F# compiler to construct the untyped syntax tree.  
+This tree is later used to reconstruct the code. The better the tree, the better Fantomas can operate on it.  
+The tree can be improved over at [dotnet/fsharp](https://github.com/dotnet/fsharp) when a valid use-case appears.
+
+### Trivia nodes
+
+Trivia nodes in the syntax tree are nodes the compiler doesn't need to compile the code. However, for Fantomas they can be the missing link to perfectly restore the code.
+
+For example, in issue [#2360](https://github.com/fsprojects/fantomas/issues/2360), there information about the `and` keyword is missing in the syntax tree.
```
Comment (dawedawe) [inline docs/docs/contributors/How Can I Contribute.md:129]:
```suggestion
For example, in issue [#2360](https://github.com/fsprojects/fantomas/issues/2360), the information about the `and` keyword is missing in the syntax tree.  
```

File: docs/docs/contributors/How Can I Contribute.md [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+### Trivia nodes
+
+Trivia nodes in the syntax tree are nodes the compiler doesn't need to compile the code. However, for Fantomas they can be the missing link to perfectly restore the code.
+
+For example, in issue [#2360](https://github.com/fsprojects/fantomas/issues/2360), there information about the `and` keyword is missing in the syntax tree.  
+If this information was available in [SynTypeDefnTrivia](../../reference/fsharp-compiler-syntaxtrivia-syntypedefntrivia.html), the bug could be fixed.
+
+### Better representation
+
+Sometimes the existing shape of the syntax tree, didn't quite cover the syntax perfectly.
```
Comment (dawedawe) [inline docs/docs/contributors/How Can I Contribute.md:134]:
```suggestion
Sometimes the existing shape of the syntax tree doesn't quite cover the syntax perfectly.
```

**Review by dawedawe — APPROVED** [review]
Nice 👍

**Comment by nojaf** [comment]
Thanks for the review!

**Pull request merged**